storybook/index.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {AppRegistry, Platform} from 'react-native';
+import {getStorybookUI, configure} from '@storybook/react-native';
+
+import {loadStories} from './storyLoader';
+
+import './rn-addons';
+
+// import stories
+configure(() => {
+    loadStories();
+}, module);
+
+// Refer to https://github.com/storybookjs/storybook/tree/master/app/react-native#start-command-parameters
+// To find allowed options for getStorybookUI
+const StorybookUIRoot = getStorybookUI({
+    port: 7007,
+    host: Platform.OS === 'ios' ? 'localhost' : '10.0.2.2',
```

Comment:
this works great with sims, but what if I run it on an actual device?

Reply:
maybe  use a ENV VAR and then fallback to this values if no env var is set ?

Reply:
I've implemented this. Its quite messy, especially when using with typescript.